erica/application/base_dto.py

```diff
@@ -1,5 +1,5 @@
-from pydantic import BaseModel
+from erica.application.grundsteuer.camel_case_input import CamelCaseInput
```

Comment:
Can we maybe move the CamelCaseInput here - it now doesn't only concern Grundsteuer anymore. Or maybe even merge BaseDto and CamelCaseInput into the same thing?

Reply:
Yes, both would make sense to me

Reply:
I thought a little bit more about it and I would prefer to keep them separate as we specifically use BaseDto for Dtos while the camelCaseInput is not that clearly only used for dtos, wdyt?

Reply:
Sure. As long as CamelCase is not hidden in Grundsteuer, I'm fine. :)